pkg/utils/meta.go

```diff
@@ -0,0 +1,21 @@
+package utils
+
+import (
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func DefaultMeta() v1.ObjectMeta {
```

Comment:
how are these generic? What does "default" mean? Default in which context and for what?

Reply:
Same as above comment. changed the function names